src/app/components/header/header.component.ts

```diff
@@ -136,4 +138,22 @@ export class HeaderComponent implements OnInit, OnDestroy {
     tokenTrackByFn(token: UserToken): string {
         return token.address;
     }
+
+    mint(token: UserToken) {
+        this.mintPending[token.address] = true;
+        const decimals = token.decimals;
+        const scaleFactor = decimals >= 18 ? 1 : decimals / 18;
```

Comment:
The test cases describe that you care about tokens with 18 decimals or 0. Are those only the cases of interest and the rest just happens but is fine? So when my imaginary token has 9 decimals, this would lead to a `scaleFactor` of 0.5. Is this intended? I just wanna make this sure.

Reply:
Yes, that's what it should do. 
There are just tests for the boundaries. I think we shouldn't overtest this as it is just a testnet functionality.

Reply:
Perfect. If that is the case I agree there that is no need to test all boundary groups etc. :heavy_check_mark: